we can use label and break with simple JavaScript blocks and functions (#2498)
javascript-tutorial/en.javascript.info: 1-js/02-first-steps/13-while-for/article.md

label: for (...)
```

> If we work with labeled loops, a call to `break/continue` is only possible from inside the loop and the label must be somewhere above the directive.

Reviewer: I would agree
just a suggestion, I'm not native English speaker:

this article is about loops, I would follow the example context "Here the call..."

```suggestion
The call to `break/continue` is only possible from inside the loop block, and the label must be somewhere above the directive.
```

Maybe "label" should be mentioned inside "structure" article.

Author: > I would agree
> just a suggestion, I'm not native English speaker:
> 
> this article is about loops, I would follow the example context "Here the call..."
> 
> Maybe "label" should be mentioned inside "structure" article.

I am not native English speaker too. But your suggestion is not clear enough. Because this concept is used very rarely. Yes, the article is about loops but this line can be more **explicit**. Or someone can add little extra explanation/warning about other usages of labels.

Reviewer: I suggested to fix only the "must be inside a loop"

But I see your point, a comprehensive turorial should note 
"break can break any block not just loops" somewhere

> If we work with labeled loops, a call to `break/continue` is only possible from inside the loop and the label must be somewhere above the directive.

Reviewer 2: ```suggestion
A call to `break/continue` for a labelled loop is only possible from inside that loop and the label must be somewhere above the directive.
```

OR

```suggestion
When working with a labelled loop, a call to `break/continue` is only possible from inside that loop and the label must be somewhere above the directive.
```
